docker/setup.sh

```diff
@@ -4,5 +4,5 @@
 git clone -b master https://github.com/hysds/multi_acquisition_localizer.git
 
 # clone ariamh to be moved to its final location by docker builder
-git clone -b standard-product https://github.com/hysds/ariamh.git
+git clone -b python3 https://github.com/hysds/ariamh.git
 
```

Comment:
Seems like we just want ariamh's isce.sh package ere to setup environment variables for isce?